cmd/version/version.go

```diff
@@ -78,8 +84,64 @@ func NewCmdVersion(rootCmd *cobra.Command) *cobra.Command {
 			}
 
 			util.TablePrinter(util.VersionColumnDefinition, rows, printers.PrintOptions{Wide: true})
+			getCliLatestVersion(rootCmd.Version)
 		},
 	}
 	cmd.SetUsageTemplate(versionCmdHelp)
 	return cmd
 }
+
+func getCliLatestVersion(currVersion string) {
+	// getting the latest version of openebsctl from sigs.k8s.io/krew-index
+	resp, err := http.Get("https://raw.githubusercontent.com/kubernetes-sigs/krew-index/master/plugins/openebs.yaml")
+	util.CheckErrDefault(err, "Error reading response body")
+	defer resp.Body.Close()
```

Comment:
so if for some reason, `err` is non-nil maybe the internet connection is down, etc the `resp` will be `nil` and the code will panic, it'd look something like a NullPointerException

Reply:
Yes, corrected.

Reply:
No I mean, something like [this](https://golangbyexample.com/resposne-body-closed-golang/) and remove the `CheckErr` function, most go-code has error handling code in the same place.

Besides, if CheckErr doesn't terminate the program there'd be a panic at line 96 because `resp` will be nil.

Reply:
i.e. `resp.Body.Close()` should be only called if err is nil, i.e. resp is non-nil, usually if err is non-nil the structure/value returned is a zero-value,

Reply:
Got it, Will do.
Actually the current version of code also works fine, since calling `util.Fatal` close the program and return thus not calling the defer after that. The program completely exits since fatala has `os.Exit()` call at the end. 
I had checked the code with no internet connection too.